docs/tutorials/google/start.ipynb

```diff
@@ -1,478 +1,532 @@
 {
```

Comment:
<p>Add another text block followed by code.</p><p> </p><p>Text: "If your quantum processor is calibrated to do a specific set of gates, you will need to transform your circuit into that gateset. For example, for a CZ gateset, you can do the following." </p><p> </p><p>Code:</p><p> </p><pre class="ql-syntax" spellcheck="false">gateset = cirq.CZTargetGateset(allow_partial_czs=True)
hw_circuit = cirq.optimize_for_target_gateset(hw_circuit, gateset=gateset)
print(hw_circuit)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Add to that text above: Other gatesets and considerations are included in the best practices guide</p><p> </p><p>with same hyperlink as Will Courtney's above.</p>

Reply:
Will follow up with another PR to add this improvement. Up to this point I've been editing the codelab's raw json file directly (this was easiest for the initial quick changes), but adding another text/code block this way will likely cause errors.

Reply:
The followup PR is #6346